Adding Url handlers to Operating System/Linux/README.md

```diff
@@ -0,0 +1,35 @@
+# Creating Custom URL Handlers on Linux:
```

Comment:
could you avoid having mixed cases and spaces in the folder/file names ... ?

Reply:
I've renamed them.